exercises/isPowerOfTwo.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"math"
+)
+
+func isWhole(n float64) bool {
+	return n == math.Round(n)
+}
+
+func isPowerOfTwo(n int) bool {
+	log := math.Log2(float64(n))
```

Comment:
Please implement this function without using `math` package (you will need a recursion).